README.md

```diff
@@ -47,22 +52,30 @@ Because a user's folder only holds items that they own, it effectively checks al
 
 ### Command line
 
-`python auditor [-r|--save_report -d|--dry -v|--verbose ITEM ...]`
+``` python
+python auditor spot [-r|--save_report -d|--dry -v|--verbose ITEM ...]
+python auditor scheduled
+```
+
+`spot`: Run a spot audit using options below.
+
+`scheduled`: Run a full audit, including sending notifications via supervisor and saving the report.
 
 Options:
 
 * `-h`, `--help`
 * `-r`, `--save_report`           Save report to the file specified in the credentials file (will be rotated)
 * `-d`, `--dry`                   Only run the checks, don't do any fixes
 * `-v`, `--verbose`               Print status updates to the console
-* ITEM                            One or more AGOL item IDs to audit. If none are specified, all items are audited.
+* `ITEM`                          One or more AGOL item IDs to audit. If none are specified, all items are audited.
 
 Example:
 
-* `auditor -v -r`
-* `auditor -v -r aaaaaaaabbbbccccddddeeeeeeeeeeee ffffffff111122223333444444444444`
+* `auditor spot -vr`
+* `auditor spot -v -r aaaaaaaabbbbccccddddeeeeeeeeeeee`
+* `auditor scheduled`
 
-### Forklift
+### Forklift (deprecated)
```

Comment:
**suggestion**: i suggest removing this and all the information for it. It will live on in the v1 branch or history.